recipes/pyutilib/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set version = "5.3.3" %}
+
+package:
+  name: pyutilib
+  version: {{ version }}
+
+source:
+  fn: PyUtilib-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/P/PyUtilib/PyUtilib-{{ version }}.tar.gz
+  md5: 2910a329844369d4caa7fc031d64ef80
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - pyutilib = pyutilib:main
+    #
+    # Would create an entry point called pyutilib that calls pyutilib.main()
+
+    - test.pyutilib=pyutilib.dev.runtests:runPyUtilibTests
+    - lbin=pyutilib.dev.lbin:main
+    - lpython=pyutilib.dev.lpython:main
+    - pypi_downloads=pyutilib.dev.pypi_downloads:main
+    - replacecopyright=pyutilib.dev.replaceCopyright:main
+    - checkcopyright=pyutilib.dev.checkCopyright:main
+    - pyutilib_test_driver=pyutilib.autotest.driver:main
+    - dispatch_srvr=pyutilib.pyro.dispatch_srvr:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose
+    - six
+
+  run:
+    - python
+    - setuptools
+    - nose
+    - six
+
+test:
+  # Python imports
+  imports:
+    - pyutilib
+    - pyutilib.autotest
+    - pyutilib.autotest.tests
+    - pyutilib.common
+    - pyutilib.component
+    - pyutilib.component.app
+    - pyutilib.component.app.tests
+    - pyutilib.component.config
+    - pyutilib.component.config.tests
+    - pyutilib.component.core
+    - pyutilib.component.core.tests
+    - pyutilib.component.executables
+    - pyutilib.component.loader
+    - pyutilib.component.loader.tests
+    - pyutilib.dev
+    - pyutilib.enum
+    - pyutilib.enum.tests
+    - pyutilib.excel
+    - pyutilib.excel.tests
+    - pyutilib.math
+    - pyutilib.math.tests
+    - pyutilib.misc
+    - pyutilib.misc.tests
+    - pyutilib.misc.tests.import_data
+    - pyutilib.misc.tests.import_data.a
+    - pyutilib.misc.tests.import_data.b
+    - pyutilib.ply
+    - pyutilib.pyro
+    - pyutilib.services
+    - pyutilib.subprocess
+    - pyutilib.subprocess.tests
+    - pyutilib.th
+    - pyutilib.th.tests
+    - pyutilib.workflow
+    - pyutilib.workflow.tests
+
+  commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+    - test.pyutilib --help
+    - pyutilib_test_driver --help
+    - dispatch_srvr --help
+
+about:
+  home: https://github.com/PyUtilib/pyutilib
+  license: BSD License
```

Comment:
What exactly should I do?  Is there a syntax you expect?

--Bill

On Tue, Apr 12, 2016 at 5:40 PM, Filipe <notifications@github.com> wrote:

> In recipes/pyutilib/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/331#discussion_r59459036>
> :
>
> > +    - pyutilib.th
> > +    - pyutilib.th.tests
> > +    - pyutilib.workflow
> > +    - pyutilib.workflow.tests
> > +
> > +  commands:
> > +    # You can put test commands to be run here.  Use this to test that the
> > +    # entry points work.
> > +
> > +    - test.pyutilib --help
> > +    - pyutilib_test_driver --help
> > +    - dispatch_srvr --help
> > +
> > +about:
> > +  home: https://github.com/PyUtilib/pyutilib
> > +  license: BSD License
>
> Cool. Can you fix that later?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/331/files/2147bdd76e61312dbfd8266a2a0087d85a579220#r59459036>
>